addons/point_of_sale/static/tests/tours/utils/product_screen_util.js

```diff
@@ -660,3 +660,27 @@ export function addDiscount(discount) {
             .flatMap((key) => Numpad.click(key)),
     ].flat();
 }
+
+export function checkVariantProductInfo(titel) {
+    //check if title and price and inventory section are present
+    return [
+        {
+            content: "Check if modal title contains title",
+            trigger: ".modal-title.text-break.flex-grow-1:contains('Office Combo')",
+            run: function () {
+                const titleElement = document.querySelector(".modal-title.text-break.flex-grow-1");
+                const titleText = titleElement.textContent.trim();
+                return titleText.includes(titel);
+            },
```

Comment:
So we want to check if the title of the modal has both `title` and `price`, we can then pass both of them as arguments, and check for them in the `trigger` with 2 `:contains` ? Something like the below

```suggestion
export function checkVariantProductInfo(title, price) {
    return [
        {
            content: "Check if modal title contains title",
            trigger: `.modal-title.text-break.flex-grow-1:contains(${title}):contains(${price})`,
        },
```

Reply:
about the function, I was not sure if I should added or not, but at the same time I did not want to do a big change in `ProductComboPriceTaxIncludedTour` that's why, but you're right its not generic